npm/util/util.go

```diff
@@ -0,0 +1,42 @@
+package util
```

Comment:
Can this util package be brought out of npm and at azure-container-networking scope? That way we can build this util and can use it everywhere within azure-container-networking. If easily possible, otherwise no worries.

Reply:
Nice suggestion. I'll do it later. But can we check this in first? This is pretty important :D